jitbuilder/optimizer/JBOptimizer.cpp

```diff
@@ -117,11 +117,12 @@ static const OptimizationStrategy JBwarmStrategyOpts[] =
    { OMR::treeSimplification                                                       },
    { OMR::trivialDeadTreeRemoval,                    OMR::IfEnabled                },
 
-   { OMR::inductionVariableAnalysis,                 OMR::IfLoops                  },
-   { OMR::loopCanonicalization,                      OMR::IfLoops                  },
+   { OMR::basicBlockOrdering                                                       }, // clean up block order going into canonicalizer
+   { OMR::loopCanonicalization,                      OMR::IfLoops                  }, // canonicalizer first, otherwise indvar data gets messed up
```

Comment:
Do you mean I should refer to it as "canonicalization" ? or should I remove the first one (since it's not specific to having loops ?)